cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -197,6 +227,20 @@ func (o *CreateOptions) Complete(clientGetter util.ClientGetter, cmd *cobra.Comm
 
 	o.PrintObj = printer.PrintObj
 
+	o.Config, err = clientGetter.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	err = setKubernetesDefaults(o.Config)
+	if err != nil {
+		return err
+	}
+
+	if o.PodRunningTimeout <= 0 {
+		return errors.New("--pod-running-timeout must be higher than zero")
+	}
```

Comment:
I'm wondering that cobra not validate it?

Reply:
What do you mean?

Reply:
We can resolve it. It is not relevant.